AlgoStoreData.cs/Fixtures/PartialFixtureCreateDeleteMethods.cs

```diff
@@ -120,5 +124,50 @@ public partial class AlgoStoreTestDataFixture : BaseTest
             return responces;
         }
 
+        public async Task<WalletDTO> CreateTestWallet()
+        {
+
+            string url = ApiPaths.WALLETS_BASE_PATH + "/hft";
+
+            WalletCreateDTO newWallet = new WalletCreateDTO()
+            {
+                Name = Helpers.Random.Next(1000, 9999).ToString() + GlobalConstants.AutoTest,
```

Comment:
Add "TestWallet " at the start of Name and Description to make it easier to distinguish that this is a wallet..